datamade/la-metro-councilmatic
# Revise packet creation to integrate with dashboard-based merger

## Overview

This PR:

- Updates `_merge_docs` to trigger a `compile_packet` DAG with the relevant values for identifier and attachment links.
- Revises requests for merged documents to hit S3 directly, instead of our proxy file server.
  - Adds a data migration to re-save all packets to update their URLs accordingly.
- Replaces `MERGER_BASE_URL` with two settings: `MERGE_HOST` and `MERGE_ENDPOINT`.

### Checklist


## Testing Instructions

Performed by @hancush:

 * Manually deploy the app and dashboard to staging, and build/push a staging tag for the app.
 * Shell into the server and save the 50 most recent event packets.
   * 42 of the 50 packets built successfully, with most of the errors being transient 404s we can address by telling scrapelib to retry 404s. Outstanding issues:
     *  https://github.com/datamade/councilmatic-document-merger/issues/1
     * https://github.com/datamade/councilmatic-document-merger/issues/2
     * https://github.com/datamade/councilmatic-document-merger/issues/3
 * Download the generated packets and confirm they look ok.

Handles #738
